code/modules/shuttle/shuttle.dm

```diff
@@ -645,20 +645,18 @@
 				var/obj/mecha/mech = AM
 				if(mech.occupant)
 					INVOKE_ASYNC(mech, TYPE_PROC_REF(/obj/mecha, get_out_and_die))
+					// SS220 PRIME EDIT - START
+					continue // It's required to avoid qdeling of mech in case of space turf. Non space turf are handled by get_out_and_die() proc
+					// SS220 PRIME EDIT - END
```

Comment:
```suggestion
					continue // SS220 EDIT: It's required to avoid qdeling of mech in case of space turf. Non space turf are handled by get_out_and_die() proc
```